cleanlab/datalab/internal/issue_manager/underperf_group.py

```diff
@@ -67,6 +81,10 @@ def __init__(
         min_cluster_samples: int = 5,
         **_: Any,
     ):
+        """
+            Note: The `min_cluster_samples` argument should not be confused with the
+        `min_samples` argument of sklearn.cluster.DBSCAN.
+        """
```

Comment:
This note belongs in the "class" docstring (the docstring that follow the `class UnderperformingGroupIssueManager` definition.
It's used instead of the __init__ method docstring when rendering the docs.